gestorpsi/careprofessional/models.py

```diff
@@ -200,57 +220,73 @@ class StudentProfile(models.Model):
     class_duration = models.CharField(_('Student Class Duration'), max_length=255, null=True, blank=True)
     register_number = models.CharField(_('Student Register Number'), max_length=255, null=True, blank=True)
     professional = models.OneToOneField('CareProfessional')
-    
+
     def __unicode__(self):
-        return u'%s' % ( self.professional )
+        return u'%s' % (self.professional)
 
     def revision(self):
-        return reversion.get_for_object(self).order_by('-revision__date_created').latest('revision__date_created').revision
+        return reversion.get_for_object(self).order_by('-revision__date_created').latest(
+            'revision__date_created').revision
+
 
 reversion.register(StudentProfile)
 
+
 class CareProfessionalManager(models.Manager):
     def active(self, organization):
-        return super(CareProfessionalManager, self).get_query_set().filter(active=True, studentprofile__id__isnull=True, person__organization = organization).order_by('person__name')
+        return super(CareProfessionalManager, self).get_query_set().filter(active=True, studentprofile__id__isnull=True,
+                                                                           person__organization=organization).order_by(
+            'person__name')
 
     def active_all(self, organization):
-        return super(CareProfessionalManager, self).get_query_set().filter(active=True, person__organization = organization).order_by('person__name')
+        return super(CareProfessionalManager, self).get_query_set().filter(active=True,
+                                                                           person__organization=organization).order_by(
+            'person__name')
 
     def deactive(self, organization):
-        return super(CareProfessionalManager, self).get_query_set().filter(active=False, studentprofile__id__isnull=True, person__organization = organization).order_by('person__name')
+        return super(CareProfessionalManager, self).get_query_set().filter(active=False,
```

Comment:
Please be more consistent when breaking big lines into small ones.
Also try to reduce them into 80 characters per line.
This is an example on how it can be:
```
    def deactive(self, organization):
        return super(CareProfessionalManager, self).get_query_set() \
        .filter(active=False,
                studentprofile__id__isnull=True,
                person__organization=organization).order_by(
                'person__name')
```